Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Traceback (most recent call last):
  File "/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py", line 84, in <module>
    preds['name_2'] = preds.apply(lambda x: find_name(x['idGen2']), axis=1)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/frame.py", line 8827, in apply
    return op.apply().__finalize__(self, method="apply")
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/apply.py", line 727, in apply
    return self.apply_standard()
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/apply.py", line 851, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/apply.py", line 867, in apply_series_generator
    results[i] = self.f(v)
  File "/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py", line 84, in <lambda>
    preds['name_2'] = preds.apply(lambda x: find_name(x['idGen2']), axis=1)
  File "/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py", line 77, in find_name
    name = map[map['description'].str.contains(id)].name
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 125, in wrapper
    return func(self, *args, **kwargs)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/core/strings/accessor.py", line 1214, in contains
    if regex and re.compile(pat).groups:
  File "/cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/re.py", line 251, in compile
    return _compile(pattern, flags)
  File "/cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.10.2/lib/python3.10/re.py", line 302, in _compile
    raise TypeError("first argument must be string or compiled pattern")
TypeError: first argument must be string or compiled pattern
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py:77: UserWarning: This pattern is interpreted as a regular expression, and has match groups. To actually get the groups, use str.extract.
  name = map[map['description'].str.contains(id)].name
/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py:77: UserWarning: This pattern is interpreted as a regular expression, and has match groups. To actually get the groups, use str.extract.
  name = map[map['description'].str.contains(id)].name
Merge completed. Merged data written to '3_merged.gff3'.
Merge completed. Merged data written to '3_merged.gff3'.
Traceback (most recent call last):
  File "/lustre06/project/6003138/rezvank/6_test/3_modify_gff3.py", line 61, in <module>
    preds = pd.read_csv(preds_file, sep='\t')
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 680, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1217, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "/home/rezvank/.local/lib/python3.10/site-packages/pandas/io/common.py", line 789, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '../../operons/data_odb/txid6239/om_outputs/pairs.csv'
